app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/ImportationFilesManager.kt

```diff
@@ -190,9 +189,13 @@ class ImportationFilesManager @Inject constructor(
         return getColumnIndex(column).takeIf { it != -1 }
     }
 
-    private suspend fun reportFailedImportation(file: PickedFile) {
-        SentryLog.e(TAG, "Failed importation of ${file.uri}")
-        _failedFiles.emit(file)
+    private fun reportFailedImportation(file: PickedFile, throwable: Throwable) {
+        SentryLog.e(TAG, "Failed importation of ${file.uri}", throwable)
+
+        //TODO: Make more precise error messages (especially for the low storage issue).
+        val errorMessage = appContext.getString(R.string.cant_import_file_x, file.fileName)
```

Comment:
Is this worth create a card for so we can prioritize it and maybe do it one day?

Reply:
Good idea, just did it.